src/content/en/2020/javascript.md

```diff
@@ -208,33 +216,53 @@ Of those external scripts, only 12.2% of them are loaded with the `async` attrib
   sql_file="breakdown_of_scripts_using_async_defer_module_nomodule.sql"
 ) }}
 
+{# TODO(authors): Why is it misleading? #}
```

Comment:
@tkadlec

Reply:
Misleading was the wrong word choice. inflated. I clarified in an edit, but basically, this means that not all of those 6% of requests get the full benefits of defer, because of that async/defer anti-pattern.